sources/packages/api/src/database/entities/student.model.ts

```diff
@@ -54,4 +54,22 @@ export class Student extends RecordDataModel {
     referencedColumnName: ColumnNames.ID,
   })
   user: User;
+
+  @Column({
+    name: "pd_date_sent",
+    nullable: true,
+  })
+  StudentPDSentAt?: Date;
+
+  @Column({
+    name: "pd_date_update",
+    nullable: true,
+  })
+  StudentPDUpdateAt?: Date;
+
+  @Column({
+    name: "pd_status",
```

Comment:
What is the difference between pd_verified and pd_status?

Reply:
pd_verified is from sfas DB and pd_status is from atbc

Reply:
Sorry for the confusion, we can assume that we will use 1 status column, no need for a 2nd. pd_verified that comes from sfasDB does actually come from ATBC. So 1 field should be fine. Thoughts?

Reply:
okay, that make sense